### DISCUSSION TITLE
Ascending party list.

### DISCUSSION THREAD

**Discussion by Watercrown** [tirem/XIUI#14] [Ideas]
One thing I'm very much looking to do is obscuring the default party window. It's tough to do currently since party members get added to the party window beneath the player. Is there any way we could have the option to have them added to the top?

**Comment by AzrielltheWise** [comment]
This is something I have also noticed and hope can be resolved.  As people are added to the party and Party List expands, it grows downward.  This means that as the party shrinks the player's name (and the remaining party members) are left floating instead of being anchored at the bottom of the screen.
